pkg/controller/csr_check.go

```diff
@@ -131,20 +133,30 @@ func authorizeCSR(
 	req *certificatesv1.CertificateSigningRequest,
 	csr *x509.CertificateRequest,
 	ca *x509.CertPool,
-) error {
+) (bool, error) {
 	if req == nil || csr == nil {
-		return fmt.Errorf("Invalid request")
+		klog.Errorf("authorizeCSR invalid request")
+		return false, nil
 	}
 
 	if isNodeClientCert(req, csr) {
-		return authorizeNodeClientCSR(c, config, machines, req, csr)
+		if config.NodeClientCert.Disabled {
+			klog.Errorf("%v: CSR rejected as the flow is disabled", req.Name)
```

Comment:
Do we want to point out here we have identified it as a node client cert? What are the security implications of logging that?

Reply:
There are no security implications.